nikola/data/themes/base/templates/base.tmpl

```diff
@@ -20,7 +20,13 @@ ${template_hooks['extra_head']()}
          ${footer.html_footer()}
     </div>
     ${base.late_load_js()}
+    % if use_cdn:
+        <script src="//cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.8.2/baguetteBox.min.js"></script>
```

Comment:
This keeps it protocol-relative, is https better? It used to give a mixed-content warning :-P